app/Http/Controllers/PokerController.php

```diff
@@ -1,133 +1,133 @@
+        public function createRoom(Request $request){
 
-        $room_id = $this->generateRandomString(); //todo make sure this is unique in db        
+            $room_id = $this->generateRandomString(); //todo make sure this is unique in db        
 
-        $now = date('Y-m-d H:i:s');
-        $id = DB::table('poker_rooms')->insertGetId([
-            'room_id' => $room_id,
-            'password' => $request->password,
-            'is_open' => true,
-            'created_at' => $now,
-            'updated_at' => $now
-        ]);
-
-        $one_week_ago = date("Y-m-d H:i:s", strtotime("-1 week"));
+            $now = date('Y-m-d H:i:s');
+            $id = DB::table('poker_rooms')->insertGetId([
+                'room_id' => $room_id,
+                'password' => $request->password,
+                'is_open' => true,
+                'created_at' => $now,
+                'updated_at' => $now
+            ]);
 
-        DB::table('poker_rooms')->where('created_at', '<=', $one_week_ago)->update([
-            'is_open' => 0
-        ]);
-        return response()->json(['success' => true, 'room_id' => $room_id]);
-    }
+            $one_week_ago = date("Y-m-d H:i:s", strtotime("-1 week"));
 
-    function getData(Request $request){
-        $room = DB::table('poker_rooms')->where('room_id', $request->room_id)->first();
-        $players = DB::table('poker_players')->where('room_id', $room->id)->get();
-        return response()->json(['room' => $room, 'players' => $players]);
-    }
-
-    function getRooms(){
-        $rooms = DB::table('poker_rooms')->get();
+            DB::table('poker_rooms')->where('created_at', '<=', $one_week_ago)->update([
+                'is_open' => 0
+            ]);
+            return response()->json(['success' => true, 'room_id' => $room_id]);
+        }
 
-        foreach($rooms as $room){
-            $room->password = $room->password != null;
+        function getData(Request $request){
+            $room = DB::table('poker_rooms')->where('room_id', $request->room_id)->first();
+            $players = DB::table('poker_players')->where('room_id', $room->id)->get();
+            return response()->json(['room' => $room, 'players' => $players]);
         }
-        return response()->json(['rooms' => $rooms]);
-    }
 
-    function passwordCheck(Request $request){
-        $room = DB::table('poker_rooms')->where('room_id', $request->room_id)->first();
+        function getRooms(){
+            $rooms = DB::table('poker_rooms')->get();
 
-        return response()->json($room->password != null);
-    }
+            foreach($rooms as $room){
+                $room->password = $room->password != null;
+            }
+            return response()->json(['rooms' => $rooms]);
+        }
 
-    function authenticate(Request $request){
-        $room = DB::table('poker_rooms')->where('room_id', $request->room_id)->first();
+        function passwordCheck(Request $request){
```

Comment:
any reason this cant return a boolean instead of a json of a boolean ?

nit: its best practice to rename boolean functions this way "hasPassword" "isAuthenticated" etc.

Reply:
I personally like having all my responses in laravel structured like this (response()->json()). in case I need to someday add variables to my returned response its very simple, and the front end its already expecting json. Would it be better to just return a boolean, and then make the necessary changes if I decide to add data to my response?

Reply:
Yes, there is currently no need to over engineer this to solve for a problem we dont have. Should changes be made, it will be very evident and easy to simply dump it to a json